dataactbroker/handlers/fileHandler.py

```diff
@@ -949,16 +949,15 @@ def check_detached_generation(self, job_id=None):
 
         return JsonResponse.create(StatusCode.OK, response_dict)
 
-    def checkGeneration(self, submission_id=None, file_type=None):
+    def checkGeneration(self):
         """ Return information about file generation jobs
 
         Returns:
             Response object with keys status, file_type, url, message.
             If file_type is D1 or D2, also includes start and end.
         """
         sess = GlobalDB.db().session
-        if submission_id is None or file_type is None:
-            submission_id, file_type = self.get_request_params_for_generate()
+        submission_id, file_type = self.get_request_params_for_generate()
```

Comment:
So how does it know which submission this is for? What if someone is generating more than one submission at a time?